audb/core/dependencies.py

```diff
@@ -350,14 +354,18 @@ def sampling_rate(self, file: str) -> int:
         return self[file][define.DependField.SAMPLING_RATE]
 
     def save(self, path: str):
-        r"""Write dependencies to CSV file.
+        r"""Write dependencies to file.
 
         Args:
-            path: path to file
+            path: path to file.
+                File extension can be ``csv`` or ``pkl``.
 
         """
         path = audeer.safe_path(path)
-        self().to_csv(path)
+        if path.endswith('csv'):
+            self().to_csv(path)
+        elif path.endswith('pkl'):
+            self().to_pickle(path, compression='xz')
```

Comment:
Maybe loading will be faster if we don't compress it?

Reply:
It is indeed much faster without compression, see https://github.com/audeering/audb/pull/38#issuecomment-828990575